.github/actions/run_docker/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # Container image that runs your code
 FROM jfalcou/compilers:v3
 
+
```

Comment:
let's be pedantic for 0.5ns.
Can we remove that noisy commit? I guess this new line is accidental.

Reply:
Being pedantic is good.

I actually placed that line there purposefully to change a Dockerfile, causing rebuilding of the docker image. I did this when I was getting those mysterious CI errors. It didn't work.

I will clean up!